SignalServiceKit/src/TSConstants.h

```diff
@@ -41,6 +41,7 @@ typedef enum { kSMSVerification, kPhoneNumberVerification } VerificationTranspor
 #define textSecureDeviceProvisioningAPIFormat @"v1/provisioning/%@"
 #define textSecureDevicesAPIFormat @"v1/devices/%@"
 #define textSecureProfileAPIFormat @"v1/profile/%@"
+#define textSecureSetProfileAPI @"v1/profile"
```

Comment:
Feel free to reimplement all the request building ceremony with a request factory any day you're bored. =)